src/test/java/org/reaktivity/specification/socks/internal/SocksFunctionsTest.java

```diff
@@ -368,7 +368,7 @@ public void shouldBuildBeginExWithIpv6AddressLeadingZeros() throws Exception
     }
 
     @Test
-    public void shouldBuildBeginExWithIpv6AddressZeroCompression1() throws Exception
+    public void shouldBuildBeginExWithIpv6AddressZeroCompression() throws Exception
```

Comment:
Suggest same naming convention as proposed for `...RouteEx...` test methods, applied to `...BeginEx...` test methods.